x/alliance/types/codec.go

```diff
@@ -2,17 +2,20 @@ package types
 
 import (
 	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/legacy"
 	"github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/types/msgservice"
+	authzcodec "github.com/cosmos/cosmos-sdk/x/authz/codec"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
 )
 
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
-	cdc.RegisterConcrete(&MsgDelegate{}, "alliance/MsgDelegate", nil)
-	cdc.RegisterConcrete(&MsgRedelegate{}, "alliance/MsgRedelegate", nil)
-	cdc.RegisterConcrete(&MsgUndelegate{}, "alliance/MsgUndelegate", nil)
-	cdc.RegisterConcrete(&MsgClaimDelegationRewards{}, "alliance/MsgClaimDelegationRewards", nil)
+	legacy.RegisterAminoMsg(cdc, &MsgDelegate{}, "alliance/MsgDelegate")
```

Comment:
think we dont need to change this. check the implementation of `RegisterAminoMsg` it uses RegisterConcrete underlying so this shouldn't matter right?